keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -284,7 +322,15 @@ _This section must be completed when targeting alpha to a release._
 * **How can this feature be enabled / disabled in a live cluster?**
   - [x] Feature gate (also fill in values in `kep.yaml`)
     - Feature gate name: MetricCardinalityEnforcement
-    - Components depending on the feature gate: All components that emit metrics
+    - Components depending on the feature gate: All components that emit metrics, i.e. (at the time of writing),
```

Comment:
The spirit of the question is: what binaries do I have to pass this feature gate to? So binary names are what we expect here.